plugins/python/redisgears_python.c

```diff
@@ -312,13 +318,15 @@ static bool PythonRequirementCtx_InstallRequirement(PythonRequirementCtx* req){
 
     int exitCode;
     if(pythonConfig.createVenv){
-        exitCode = RedisGears_ExecuteCommand(NULL, "notice", "/bin/bash -c \"source %s/bin/activate; cd '%s'; python -m pip install --no-index --disable-pip-version-check %s\"", venvDir, req->basePath, filesInDir);
+        exitCode = RedisGears_ExecuteCommand(NULL, "notice", "/bin/bash -c \"source %s/bin/activate; cd '%s'; python -m pip install --no-cache-dir --force-reinstall --no-index --disable-pip-version-check %s\"", venvDir, req->basePath, filesInDir);
     }else{
-        exitCode = RedisGears_ExecuteCommand(NULL, "notice", "/bin/bash -c \"cd '%s'; %s/bin/python3 -m pip install --no-index --disable-pip-version-check %s\"", req->basePath, venvDir, filesInDir);
+        exitCode = RedisGears_ExecuteCommand(NULL, "notice", "/bin/bash -c \"cd '%s'; %s/bin/python3 -m pip install --no-cache-dir --force-reinstall --no-index --disable-pip-version-check %s\"", req->basePath, venvDir, filesInDir);
```

Comment:
To speed up gears we should try using --compile as well

Reply:
Are we sure we want to add it to patch version? It can not break anything?

Reply:
The cython compiler is supposed to do this upon load of a module. This would reduce load times. I'm game for separating that into it's own PR.